test/bin/check_cloudwatch_composite_metric_spec.rb

```diff
@@ -31,4 +35,209 @@ def ok(*)
       expect(@check.metric_desc).to eq('AWS/EC2-numerator/denominator(dimensions)')
     end
   end
+
+  describe '#composite_check' do
+    before :each do
+      Aws.config = { stub_responses: true }
+    end
+
+    after :each do
+      Aws.config = {}
+    end
+    it 'should exit unknown if any data is nil and no flags are passed' do
+      config = {
+        namespace: 'namespace',
+        numerator_metric_name: 'numerator_metric_name',
+        denominator_metric_name: 'denominator_metric_name',
+        dimensions: 'dimensions',
+        period: 2,
+        statistics: 'stats',
+        unit: 'foo'
+      }
+      @check.config = config
+      aws_resp = @check.client.stub_data(:get_metric_statistics)
+      allow(@check).to receive(:get_metric).and_return(aws_resp)
+      allow(@check).to receive(:metric_desc).and_return('test')
+
+      resp, msg = @check.composite_check
+      expect(resp).to eq(:unknown)
+      expect(msg).to eq('test could not be retrieved')
+    end
+
```

Comment:
nitpicky but for consistency let's either put a newline before each assertion (preferable) or kill this line.

Reply:
Added new line before each assertion